usefulUtilities/hiFiCalendar/entityServerScripts/hiFiCalendarServer.js

```diff
@@ -8,119 +8,348 @@
 
 
 (function() {
+    var INTERVAL_FREQUENCY_MS = 60000;
+    var EXPIRY_BUFFER = 300000;
```

Comment:
what is `EXPIRY_BUFFER` ?

Reply:
It is the window of time before the token expires (5 minutes) in which the server should make it's request to refresh the token.